scss/components/_messages.scss

```diff
@@ -18,29 +23,29 @@
 .message {
   @include clearfix();
   background-size: 2rem;
-  background-color: rgba($ivory-dark, .5);
+  background-color: #eee6dd;
```

Comment:
Color literals like `#eee6dd` should only be used in variable declarations; they should be referred to via variable everywhere else.